Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "ya-client"
-version = "0.4.0"
+version = "0.4.1"
```

```suggestion
version = "0.4.0"
```

plss rather use `cargo release` for bumping version: @prekucki says